### PR TITLE
Updated oversampling, minor edits to reflect changes, minor edits to examples

### PR DESCRIPTION
**Pull request by LyricPants66133**
Added set and get functions for oversampling, using 5 different word options to reflect a different amount of time. I updated keywords.txt and readme to reflect these changes. I also made minor edits to the examples, and added a short explanation of the available oversampling settings. I also tested the library with my sensor, and it works very well.

### CODE REVIEW

**Comment by RobTillaart** [comment]
Thanks for the PR, I will try to look into it tomorrow in more depth. 
(busy upgrading / cleaning all my libs for open commits so it might take some time"

**Comment by RobTillaart** [comment]
quick first look seems good!,
maybe create a specific example to show the working?

**Comment by RobTillaart** [comment]
title of issue => exmaples  ?

File: MS5611.h [hunk 1]
```diff
@@ -19,6 +19,14 @@
 #define MS5611_ERROR_2                        2         // low level I2C error
 #define MS5611_NOT_READ                       -999
 
+enum osr_t
+{
+    OSR_ULTRA_HIGH = 12, // 10 millis
```
Comment (RobTillaart) [inline MS5611.h:24]: timing measured on an UNO?

**Comment by RobTillaart** [comment]
@LyricPants66133 
Typical math for oversampling is 4x as much samples gives 1 bit accuracy extra.
More info - https://en.wikipedia.org/wiki/Oversampling#Resolution

File: MS5611.cpp [hunk 2]
```diff
@@ -129,17 +130,17 @@ void MS5611::reset()
 }
 
 
-int MS5611::read(uint8_t bits)
+int MS5611::read()
 {
```
Comment (RobTillaart) [inline MS5611.cpp:134]:
```
int MS5611::read(uint8_t bits)
int MS5611::read()
```
this breaks the interface so this needs some thoughts.

Your goal is to set the oversampling rate once and just call read() without parameters.
(as far as I understand)

**Comment by RobTillaart** [comment]
to be continued ....

File: MS5611.h (hunk 1 above)
Comment (LyricPants66133) [reply MS5611.h:24]: Timings were not measured, they are the values that correspond to the del[] array in the convert function.

**Comment by RobTillaart** [comment]
@LyricPants66133 
released 0.3.2 
I fixed backwards compatibility, while keeping the functionality you proposed. 
renamed the internal variabele name to sampleRate, added some more examples 

Can you please verify it still works for you? Thanks.

**Pull request merged**